packages/commonwealth/client/scripts/views/pages/LaunchToken/steps/TokenInformationStep/TokenInformationForm/TokenInformationForm.tsx

```diff
@@ -0,0 +1,151 @@
+import useAppStatus from 'hooks/useAppStatus';
+import { useBrowserAnalyticsTrack } from 'hooks/useBrowserAnalyticsTrack';
+import React, { useState } from 'react';
+import {
+  BaseMixpanelPayload,
+  MixpanelCommunityCreationEvent,
+  MixpanelLoginPayload,
+} from 'shared/analytics/types';
+import {
+  CWCoverImageUploader,
+  ImageBehavior,
+} from 'views/components/component_kit/cw_cover_image_uploader';
+import { CWLabel } from 'views/components/component_kit/cw_label';
+import { CWTextArea } from 'views/components/component_kit/cw_text_area';
+import { CWButton } from 'views/components/component_kit/new_designs/CWButton';
+import CWCommunitySelector from 'views/components/component_kit/new_designs/CWCommunitySelector';
+import { CWForm } from 'views/components/component_kit/new_designs/CWForm';
+import { CWTextInput } from 'views/components/component_kit/new_designs/CWTextInput';
+import { openConfirmation } from 'views/modals/confirmation_modal';
+import { communityTypeOptions } from 'views/pages/CreateCommunity/steps/CommunityTypeStep/helpers';
+import './TokenInformationForm.scss';
+import { FormSubmitValues, TokenInformationFormProps } from './types';
+import { tokenInformationFormValidationSchema } from './validation';
+
+const TokenInformationForm = ({
+  onSubmit,
+  onCancel,
+}: TokenInformationFormProps) => {
+  const [isProcessingProfileImage, setIsProcessingProfileImage] =
+    useState(false);
+
+  const { isAddedToHomeScreen } = useAppStatus();
+
+  const [baseOption] = communityTypeOptions;
```

Comment:
in erc20 weighted topics and in stake we support multiple chains. 

![image](https://github.com/user-attachments/assets/ccd80ef9-5788-4598-8a62-6527b6a0a262)







Do we support only Base for launchpad?